.stylelintrc

```diff
@@ -49,7 +49,61 @@
       }
     ]
   },
+  "overrides": [
+    {
+      "files": ["./src/button/*.scss"],
+      "rules": {
+        "property-disallowed-list": [
```

Comment:
Why do we need this long list, if `csstools/use-logical` seem to enforce the same rules anyway?

Reply:
You are correct that there are redundancies between these rules and the plugin. The first reason is that the plugin is more permissive with shorthands (i.e. border, margin, padding) than we want to be instead of opting for logical only shorthands (i.e. border-inline, padding-block). The second is that, over the long term, it is likely that we change and/or remove this plugin and the rules ensure that these properties never make an appearance in our codebase. A minimal alternative to this can be seen in the parent PR: https://github.com/cloudscape-design/components/pull/1713/files#diff-a3f38f2cae79a3819f93ff1a9d4cd281cbe8f95696e14a29864f08796d3dc568

I'm inclined to use the comprehensive list but am open to an alternative opinion.

Reply:
What's stopping us from applying your change from another PR right away? The shorter list looks good for me